Desktop_Interface/isobuffer.cpp

```diff
@@ -328,15 +325,14 @@ void isoBuffer::serialManage(double baudRate, UartParity parity)
 	if (m_decoder == NULL)
 	{
 		m_decoder = new uartStyleDecoder(this);
-		// TODO: Look into using the type safe version of connect.
```

Comment:
Right? I found out about that one the other day watching talk about Qt on youtube, turns out it can catch a bunch of bugs on compile time that would otherwise be crashes (It doesn't seem to be the case here, but better be safe than sorry).